diagnostic.md

```diff
@@ -38,7 +42,9 @@ $base-margin: 1.5em;
 ```
 
 ```scss
-// your response here
+li {
+  $base-margin: $base-margin * 0.5;
+}
 ```
```

Comment:
Close! This would be redefining the `$base-margin` variable, rather than setting `margin`.

```CSS
li {
  margin: $base-margin * .5;
}
```